README.md

```diff
@@ -27,13 +29,13 @@ Users are able to use this App as a Todo-List or a Note Taking application.
         sudo npm install -g bower
         sudo npm install -g grunt-cli
 
-2. Clone this repo.
+3. Clone the repository by running: git clone https://github.com/'YourGitHubUserName'/AngularJsTodoApp.git
```

Comment:
Having "Clone this repository by running `git clone https://github.com/jaylenw/AngularJsTodoApp.git` is what I would like to be. I think users by default will be cloning specifically this repository so the username would be mine. But then I thought about it, if they forked this repository, then it would be `git clone https://github.com/'TheirGithubUsername'/AngularJsTodoApp.git` for them to clone the correct repo. To avoid confusion, I would like to keep it generic to "Clone this repository", because it depends if they cloned the repo under my username or under theirs if they forked this project. We don't want our fellow developers cloning a repository they didn't intend to accidentally.